src/js/components/Preferences/Preferences.js

```diff
@@ -46,6 +47,7 @@ export default function Preferences() {
         <form ref={setForm} className="settings-form">
           <Timezone config={prefsForm.timeZone} />
           <TimeFormat config={prefsForm.timeFormat} />
+          <ZeekRunner config={prefsForm.zeekRunner} />
```

Comment:
Almost a direct copy and paste from JSONTypeConfig. Potential for abstracting soon.

Reply:
Update: Did abstract this to a FileInput component used for the preferences modal.